src/igvfd/schemas/measurement_set.json

```diff
@@ -87,56 +87,57 @@
             "description": "The custom lab preferred label for the experiment performed in this measurement set.",
             "type": "string",
             "enum": [
```

Comment:
I reorganized this list to be based on assay not alphabetical ordering in https://github.com/IGVF-DACC/igvfd/pull/1063/files. I had brought it up with Ben and Jennifer, my reasoning was that alphabetical ordering here is not helpful and it would be more useful for the submitters if relevant assays were grouped together in this long list of entries. Say for example, a submitter is submitting "Growth CRISPR screen" but first see "Proliferation CRISPR screen" and chooses this term instead as they are unaware of the more precise term.

Reply:
It looked very strange to me - I wasn't aware we did that deliberately.